### DISCUSSION TITLE
All the text field inputs for numbers seem to be missing from the webui

### DISCUSSION THREAD

**Discussion by jmattthew** [TheLastBen/fast-stable-diffusion#457] [Q&A]
I'm using the fast-stable-diffusion Colab notebook.  It's working as expected except that I can't edit the number values that are next to the sliders.  For example, the steps, CFG, width, and height.  The sliders work, and the number value changes if I move the slider, but the numbers are static rather than inside of an editable text field.  

Normally the Automatic1111 webui has form fields to type these values in directly, which is much easier than using the sliders.  Have I made an error?  I tried running "update_repo" in the Colab notebook, but it reports "already up to date".

**Comment by TheLastBen** [comment]
I'll check it out, in the meantime you can click on the slider and use the keyboard arrow keys up and down to control the value.

**Reply by jmattthew** [reply]
Thanks!  I didn't know that I could use the arrow keys too
